### ISSUE TITLE
[bug] Inline error for privacy policy not matching design system style

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#71502]
## Value Statement
**_As a_** Veteran
**_I want to_** be able to spot and understand the error
**_So that_** I can fix it for my submission

## Background Context
Found this error on the Review your request section

### Replicate:
Try to proceed without hitting the checkbox to accept privacy policy.

### Screenshot:
<img width="640" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTlCQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0a3f76fcec38f8cede21d7c54d5dda0103d79634/Screenshot%202023-12-07%20at%202.19.49%E2%80%AFPM.png" alt="Screenshot 2023-12-07 at 2.19.49 PM.png" />

### What should happen:
- Inline error messages should be styled in red
<img width="732" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTFCQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--47b0ad07f0246a9f683a655221f6a460c71478c8/Screenshot%202023-12-07%20at%202.23.01%E2%80%AFPM.png" alt="Screenshot 2023-12-07 at 2.23.01 PM.png" />

## Tasks
- [ ] Style error message to match inline errors

**Comment by Mottie** [comment]
This is a problem with the `va-checkbox` web component - I've [opened a ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2355) for them.

**Comment by Mottie** [comment]
Fix merged into component library. Now we need to wait for the component library update before we can test.

**Comment by saderagsdale** [comment]
Waiting for the design system team to update the component library (typically happens once per sprint). @Mottie is reaching out to confirm with them.

**Comment by Mottie** [comment]
Next component library update is scheduled for Jan 8, 2024

**Comment by HeatherWidmont** [comment]
@Mottie will test this in staging late today or early tomorrow

**Comment by Mottie** [comment]
The library has not yet been updated; awaiting response with new expected date.

**Comment by Mottie** [comment]
Library is now updated. We can test this in staging.

**Comment by HeatherWidmont** [comment]
Will be tested today in the parking lot

**Comment by eileen-coforma** [comment]
Validated in parking lot
